web/app/js/components/BaseTable.test.js

```diff
@@ -0,0 +1,94 @@
+import _merge from 'lodash/merge';
+import ApiHelpers from './util/ApiHelpers.jsx';
+import BaseTable from './BaseTable.jsx';
+import { routerWrap } from '../../test/testHelpers.jsx';
+import { mount } from 'enzyme';
+
+describe("Tests for <BaseTable>", () => {
+  const defaultProps = {
+    api: ApiHelpers(""),
+  };
+
+  it("renders the table with sample data", () => {
+    let extraProps = _merge({}, defaultProps, {
+      tableRows: [{
+        name: "authors",
+        namespace: "default",
+        key: "default-deployment-authors",
+        successRate: 0.6956521739130435,
+        requestRate: 6.9,
+        latency: {P50: 4, P95: 17, P99: 66},
+        pods: {totalPods: "1", meshedPods: "1"},
```

Comment:
if we're only including two columns for the table definition, do we need all the data in this object? it kinda makes it harder to see what's being tested. maybe include only data for columns that are defined? 
another thing to do could be to have a generate fake data function that will generate this datum to avoid having a bunch of these data objects in here (mentioned below too).

Reply:
Agreed re: unused data, I removed it from the tests and will build a generate fake data function in a separate PR.